src/components/WhoWeAre/Team.tsx

```diff
@@ -33,4 +33,9 @@ const MarginTeam = styled.div`
   margin-bottom: 50px;
 `;
 
+const TeamName = styled.h3`
+  margin: 0px;
```

Comment:
You can simply use 0 and 0, rather than adding px

Reply:
removed px